src/lib/api.test.ts

```diff
@@ -0,0 +1,129 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest'
+import { managementAppGetRunnableStudiesRequest, toaGetRunsRequest } from './api'
+import jwt from 'jsonwebtoken'
+
+beforeEach(() => {
+    process.env.MANAGEMENT_APP_PRIVATE_KEY = 'mockprivatekeyvalue'
```

Comment:
🤔 @philschatz i think you left a similar comment on my test about using the jest `setupFile` feature, but i couldn't find a similar feature in vitest which we're using (i think referencing jest makes things a little more confusing since its not totally 1-1 with feature names or parity), [i ended up taking this same approach for setting up/tearing down env variables here](https://github.com/safeinsights/trusted-output-app/pull/15/files#diff-93eb310abfe57eda4f26a00fb18fb508e2df0f4d0b8fe0270928ec62c2da7068)

i figure whatever we do in one place we should do in the other too

Reply:
thanks buddies 🙏